app/src/main/java/org/oppia/app/help/HelpActivity.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.help
+
+import android.os.Bundle
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The help page activity for users FAQ and feedbacks. */
+class HelpActivity : InjectableAppCompatActivity() {
+  @Inject
+  lateinit var helpActivityPresenter: HelpActivityPresenter
```

Comment:
Ditto: This should be in a single line.
`@Inject lateinit var helpActivityPresenter: HelpActivityPresenter`

Reply:
ditto